lib/wor/paginate/adapters/helpers/total_count.rb

```diff
@@ -0,0 +1,17 @@
+module Wor
+  module Paginate
+    module Adapters
+      module Helpers
+        module TotalCount
+          def total_count
+            content = @content.reorder(nil)
+            content_size = content.try(:size)
```

Comment:
[#try method](https://apidock.com/rails/Object/try) return nil if object doesn't respond to arg method... What do you think about to use #public_send method instead of try?